narwhals/_arrow/series.py

```diff
@@ -922,6 +922,23 @@ def rolling_sum(
             result = result[offset_left + offset_right :]
         return result
 
+    def rolling_mean(
+        self: Self,
+        window_size: int,
+        *,
+        min_periods: int | None,
+        center: bool,
+    ) -> Self:
+        rolling_sum = self.rolling_sum(
+            window_size=window_size, min_periods=min_periods, center=center
+        )
+        rolling_count = (
+            (~self.is_null())
+            .cast(self._dtypes.Int32())
+            .rolling_sum(window_size=window_size, min_periods=min_periods, center=center)
+        )
+        return rolling_sum / rolling_count
```

Comment:
This is a lazy way of doing it. Performances can be enhanced with the same routine of `rolling_sum`, and dividing by the count at the end. I will address it